Ryujinx/Ui/RendererWidgetBase.cs

```diff
@@ -84,8 +87,10 @@ public RendererWidgetBase(InputManager inputManager, GraphicsDebugLevel glLogLev
             _inputManager = inputManager;
             _inputManager.SetMouseDriver(mouseDriver);
             NpadManager = _inputManager.CreateNpadManager();
+            NpadManager.ComboPressed += OnComboPressed;
             TouchScreenManager = _inputManager.CreateTouchScreenManager();
```

Comment:
```suggestion
            _inputManager = inputManager;
            
            _inputManager.SetMouseDriver(mouseDriver);

            NpadManager = _inputManager.CreateNpadManager();
            TouchScreenManager = _inputManager.CreateTouchScreenManager();
            
            NpadManager.ComboPressed += OnComboPressed;
           
```
Ideally `_inputManager` would also be above `mouseDriver`.